### PR TITLE
Fix a logic template in Algebroids and adjust to homalg-project/FinSetsForCAP#193

### PR DESCRIPTION
**Pull request by zickgraf**

### CODE REVIEW

**Comment by zickgraf** [comment]
Note: This is not complete, that is, not all objects in FinSets which are created in the compiled code are created as big integers. This would require adjusting more logic template to use big integers (e.g. `Sum( [ ] )` should maybe be translated to `BigInt( 0 )` instead of simply `0`). But since in CAP there is no difference between integers and big integers, this can come later.

File: FunctorCategories/gap/precompiled_categories/FinBouquetsPrecompiled.gi [hunk 1]
```diff
@@ -225,7 +227,7 @@ function ( cat_1, source_1, range_1, alpha_1 )
     deduped_60_1 := deduped_64_1[3];
     deduped_59_1 := deduped_64_1[2];
     deduped_58_1 := CreateCapCategoryObjectWithAttributes( deduped_62_1, Length, 2 );
```
Comment (mohamed-barakat) [inline FunctorCategories/gap/precompiled_categories/FinBouquetsPrecompiled.gi:229]: Why isn't this `BigInt( 2 )`?

Comment (mohamed-barakat) [reply FunctorCategories/gap/precompiled_categories/FinBouquetsPrecompiled.gi:229]: and at several other places as well.

**Review by mohamed-barakat — CHANGES_REQUESTED** [review]

File: FunctorCategories/gap/precompiled_categories/FinBouquetsPrecompiled.gi (hunk 1 above)
Comment (zickgraf) [reply FunctorCategories/gap/precompiled_categories/FinBouquetsPrecompiled.gi:229]: See my comment above: https://github.com/homalg-project/CategoricalTowers/pull/185#issuecomment-1516664502

File: Algebroids/gap/precompiled_categories/CategoryFromNerveDataPrecompiled.gi [hunk 2]
```diff
@@ -358,11 +359,11 @@ function ( cat_1, alpha_1, beta_1 )
...
-                           local deduped_1_2;
-                           deduped_1_2 := (1 + CAP_JIT_INCOMPLETE_LOGIC( logic_new_func_x_2 ));
-                           return (-1 + SafePosition( deduped_10_1, (hoisted_7_1[deduped_1_2] + hoisted_8_1[deduped_1_2] * deduped_11_1) ));
-                       end ), CAP_JIT_INCOMPLETE_LOGIC( [ 0 .. (Length( deduped_10_1 ) - 1) ][SafePosition( deduped_10_1, (AsList( MapOfMorphism( alpha_1 ) )[1] + AsList( MapOfMorphism( beta_1 ) )[1] * deduped_11_1) )] ) ))] ] ) );
+    return CreateCapCategoryMorphismWithAttributes( cat_1, Source( alpha_1 ), Range( beta_1 ), MapOfMorphism, CreateCapCategoryMorphismWithAttributes( deduped_17_1, CreateCapCategoryObjectWithAttributes( deduped_17_1, Length, deduped_18_1 ), Range( deduped_13_1 ), AsList, [ AsList( deduped_13_1 )[1 + (-1 + BigInt( SafePosition( List( [ 0 .. (Length( deduped_19_1[1][3] ) - 1) ], function ( logic_new_func_x_2 )
```
[the remark this answers is not in the document]
Comment (mohamed-barakat) [reply Algebroids/gap/precompiled_categories/CategoryFromNerveDataPrecompiled.gi:362]: Now I see it. Thanks.

**Review by mohamed-barakat — APPROVED** [review]

**Pull request merged**